cmd/frieza/cli_snapshot.go

```diff
@@ -81,12 +81,11 @@ func snapshotNew(customConfigPath string, args []string) {
 	if len(customConfigPath) > 0 {
 		configPath = &customConfigPath
 	}
-	config, err := ConfigLoad(configPath)
+	config, err := ConfigLoadWithDefault(configPath)
 	if err != nil {
 		log.Fatalf("Cannot load configuration: %s", err.Error())
 	}
-
-	if _, err = SnapshotLoad(snapshotName); err == nil {
+	if _, err = SnapshotLoad(snapshotName, config.SnapshotFolderPath); err == nil {
 		log.Fatalf("Snapshot %s already exist", snapshotName)
```

Comment:
After this, the folder path is not stored in the definition of the variable `snapshot` (l 120)

Reply:
Sure, SnapshotFolderPath from snapshot struct should probably be removed as global config stores it